joomla/joomla-cms
# [4.0] Accessibility Highlight Links Inconsistent

Pull Request for Issue #29292

### Summary of Changes

Removed unwanted class

### Testing Instructions

Steps to reproduce the issue
Login to admin and set "highlight links" in your Accessibility Settings to yes.
Click Save.
Note that there is a red box around all three icons at top

### Actual result BEFORE applying this Pull Request

<img width="312" alt="83340240-903af600-a2cd-11ea-9d2a-52fcd78398ed" src="https://user-images.githubusercontent.com/19772406/98436519-ec6a3780-2101-11eb-9a18-f36532ed11f3.png">

### Expected result AFTER applying this Pull Request

<img width="311" alt="83340208-50740e80-a2cd-11ea-8f9a-460450f46cc1" src="https://user-images.githubusercontent.com/19772406/98436516-e5432980-2101-11eb-91a7-44d17026e720.png">

### Documentation Changes Required
